web/src/components/Upstream/UpstreamForm.tsx

```diff
@@ -663,6 +663,9 @@ const UpstreamForm: React.FC<Props> = forwardRef(
                   name: formatMessage({ id: 'page.upstream.step.select.upstream.select.option' }),
                   id: '',
                 },
+                {
+                  name: formatMessage({ id: 'page.upstream.step.select.upstream.select.none' }),
```

Comment:
Hii @stu01509, thanks for the good work :+1: . I'm just curious a bit about the changes. The generic component UpstreamForm is used by services as we as other two pages
![image](https://user-images.githubusercontent.com/41498427/111871855-39b0bc80-89b2-11eb-8cd3-9b1eab00fea5.png)
which includes routes & upstream too. Making a global change to the component itself will let others select `None` which is not solicited (because route/upstream with none value doesn't make sense). I may be wrong though, haha.

Hey, guys please confirm.

Reply:
Hi @bisakhmondal 

Thanks for your mention, I think it may need to wait for @juzhiyuan and @Firstsawyou confirm haha.